From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 18 May 2019 12:19:35 +0200
Subject: [PATCH] {MAYBE UPS} samples/vfs/test-statx: remove unused attribute

statx is used, and clang does not like it:

    samples/vfs/test-statx.c:199:16: warning: implicit conversion
    changes signedness: 'int' to 'unsigned int' [-Wsign-conversion]
        for (offset = from; offset < to; offset++) {
                    ~ ^~~~
---
 samples/vfs/test-statx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/samples/vfs/test-statx.c b/samples/vfs/test-statx.c
index 613e28adffef..ae4ee330f2eb 100644
--- a/samples/vfs/test-statx.c
+++ b/samples/vfs/test-statx.c
@@ -37,7 +37,7 @@ struct foo { struct statx s; };
 #define __NR_statx -1
 #endif
 
-static __attribute__((unused))
+static
 ssize_t statx(int dfd, const char *filename, unsigned flags,
 	      unsigned int mask, struct statx *buffer)
 {
-- 
